thinkhazard/templates/report.jinja2

```diff
@@ -79,7 +79,7 @@ Think Hazard - {{ division.name}}
           <h2 class="page-header detail">
             {{ current_hazard.hazardtype.title }}
             <small>
-              Hazard level: <span class="level">{{ current_hazard.categorytype.title }}</span>
+              Hazard level: <span class="level hazard-active">{{ current_hazard.categorytype.title }}</span>
```

Comment:
I added this class thinking it might be help full after when I change Item 5 the disabled hazard section, they also have the label class but a different font-style. I will see if I can do without it.